Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV GOPATH /go
 ENV PATH ${GOPATH}/bin:/usr/local/go/bin:$PATH
 ENV RMAPIREPO github.com/juruen/rmapi
 
-RUN go get -u ${RMAPIREPO}
+RUN git clone https://github.com/juruen/rmapi && cd rmapi && go install
```

Comment:
Why not use the `RMAPIREPO` variable above?

Reply:
Yes, you could - to be elegant. Will modify it now.